@xen-orchestra/web/src/jobs/args.ts

```diff
@@ -0,0 +1,12 @@
+import type { XoNewVmData } from '@/types/xo/new-vm.type.ts'
+import { defineJobArg } from '@core/packages/job'
+
+export const vmArg = defineJobArg({
+  identify: (vm: XoNewVmData) => vm.name_label,
```

Comment:
What `identify` used for?
I found that weird to `idenfity` a VM from his `name_label` (since is not a unique value)